qml/DeviceManagerViewer.qml

```diff
@@ -16,7 +16,18 @@ PingPopup {
     }
 
     Component.onCompleted: {
-        open()
+        if(!DeviceManager.primarySensor) {
```

Comment:
Will this interfere with manual connections where the sensor does not immediately respond? ie. if you wanted to connect manually to /dev/ttyUSB0 to do a ping1d recovery with the stm32 bootloader.

Reply:
No, primary sensor is a pointer for the connected device, if there is something connected the is valid.